Web/Components/Pages/Login.razor.css

```
@@ -0,0 +1,74 @@
+::deep .landing-background {
+    min-height: 100vh;
+    width: 100vw;
+    background-size: cover;
+    background-position: center;
+    position: fixed;
+    top: 0; left: 0;
+    z-index: -1;
+}
+
+::deep .fixed-login-column {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)